src/core/pathname.cc

```diff
@@ -911,6 +911,12 @@ CL_DEFUN Pathname_sp cl__pathname(T_sp x) {
   return gc::As<Pathname_sp>(x);
 }
 
+CL_DOCSTRING("Returns the pathname denoted by pathspec.")
```

Comment:
please put in a comment or something here and on cl__truename explaining the odd definition (that these will be made generic later)